### PR TITLE
CP-7238 - add snowflake unload data script

### PR DESCRIPTION
**Pull request by walles2**
This is taking the script use to unload snowflake data and put it here. Most of the review is about the placement of this file - I added a new scripts directory, and a snowflake directory within that. If there's a better place to put this, I'd gladly put it there, but this is what made sense to me.

### CODE REVIEW

**Comment by walles2** [comment]
updated the s3 version

**Comment by walles2** [comment]
I'm going to make this location the source of truth for this script - once this pr is in, I'll update the wiki pages to point to the script in this repo.

**Comment by wlblanchette** [comment]
> I'm going to make this location the source of truth for this script - once this pr is in, I'll update the wiki pages to point to the script in this repo.

@muellerk22 FYI. We're updating the snowflake unload script.

File: scripts/snowflake/unload_billing_data.sql [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+USE ROLE IDENTIFIER($CLOUDZERO_BILLING_ACCESS_SCRIPT_ROLE);
+CREATE STORAGE INTEGRATION IF NOT EXISTS IDENTIFIER($CLOUDZERO_BILLING_ACCESS_STORAGE_INTEGRATION)
+TYPE = EXTERNAL_STAGE
+  STORAGE_PROVIDER = S3
+  ENABLED = TRUE
+  STORAGE_AWS_ROLE_ARN = $CLOUDZERO_BILLING_ACCESS_STORAGE_AWS_ROLE_ARN
+STORAGE_ALLOWED_LOCATIONS = ('*');
+
+DESC STORAGE INTEGRATION IDENTIFIER($CLOUDZERO_BILLING_ACCESS_STORAGE_INTEGRATION);
+-- NOTE: Must update the IAM Role with STORAGE_AWS_IAM_USER_ARN and STORAGE_AWS_EXTERNAL_ID
```
Comment (wlblanchette) [inline scripts/snowflake/unload_billing_data.sql:31]: Are these comments obsolete? What is the shape of the change being made to this script? It's hard to tell without a diff

**Review by mattyellen — APPROVED** [review]

**Review by wlblanchette — APPROVED** [review]

**Pull request merged**